src/python/pants/backend/java/target_types.py

```diff
@@ -29,6 +31,13 @@ class JavaSourceField(SingleSourceField):
 
 class JavaGeneratorSourcesField(MultipleSourcesField):
     expected_file_extensions = (".java",)
+    help = generate_multiple_sources_field_help_message(
+        softwrap(
+            """
+            Example: `sources=['Example.java', '*Test.java', '!TestIgnore.java']`.
+            """
+        )
+    )
```

Comment:
It's probably better to have a distinct example on `java_sources` vs `junit_tests`, rather than on this helper class.

Reply:
Done!